### PR TITLE
Refactoring/fairspc 82

### PR DESCRIPTION
**Pull request by tgreenwood**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2462,8 +2462,8 @@ use `--set-file` option:
 +
 This should also restart the Saturn pod. If not, trigger the restart manually.
 +
-For local development - replace vocabulary file in link:projects/saturn/vocabulary.ttl[projects/saturn/vocabulary.ttl]
-and views configuration in link:projects/saturn/views.ttl[projects/saturn/views.ttl].
+For local development - replace vocabulary file in link:projects/saturn/src/main/resources/vocabulary.ttl[vocabulary.ttl]
+and views configuration in link:projects/saturn/src/main/resources/views.yaml[views.ttl].
```
Comment (ewelinajozwik) [inline README.adoc]:
```suggestion
and views configuration in link:projects/saturn/src/main/resources/views.yaml[views.yaml].
```

File: projects/mercury/package.json [hunk 2]
```diff
@@ -4,11 +4,12 @@
   "private": true,
   "scripts": {
     "dev": "yarn build && concurrently \"yarn localdevelopment\" \"yarn saturn\" \"yarn pluto\" \"yarn start\"",
+    "dev-no-saturn": "yarn build && concurrently \"yarn localdevelopment\" \"yarn pluto\" \"yarn start\"",
```
Comment (ewelinajozwik) [inline projects/mercury/package.json]: Why is it needed? It looks like we started adding all possible combinations here... I would rather keep the script options simple (I already didn't like the dev-be and dev-fe separation)

File: projects/saturn/build.gradle [hunk 3]
```diff
@@ -38,41 +40,39 @@ repositories {
...
-    implementation 'org.keycloak:keycloak-jetty94-adapter:23.0.7'
     implementation 'org.keycloak:keycloak-admin-client:23.0.7'
     implementation 'org.keycloak:keycloak-policy-enforcer:23.0.7'
 
     implementation "org.postgresql:postgresql:${postgresqlVersion}"
     implementation "com.zaxxer:HikariCP:5.1.0"
 
-    runtimeOnly 'org.apache.logging.log4j:log4j-api:2.23.0'
-    runtimeOnly 'org.apache.logging.log4j:log4j-core:2.23.0'
-    runtimeOnly 'org.apache.logging.log4j:log4j-slf4j2-impl:2.23.0'
```
Comment (ewelinajozwik) [inline projects/saturn/build.gradle:74]: why is it not needed anymore?

Comment (tgreenwood) [reply projects/saturn/build.gradle:74]: log4j-slf4j2-impl is not used, all the rest come with other dependencies

by the way, I noticed you use several different log providers in Saturn (slf4j, lo4j, log4j2) - I will create ticket to align it across Saturn

File: projects/mercury/package.json (hunk 2 above)
Comment (tgreenwood) [reply projects/mercury/package.json]: I accidentally committed it, used locally as I need only saturn to run in debug, I removed it

File: projects/saturn/src/main/resources/application.yaml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+      resource-id: workspace-client
+      principal-attribute: preferred_username
+
+application:
+  publicUrl: ${PUBLIC_URL:http://localhost:8080}
+  jena:
+    # Base IRI for all metadata entities
+    metadataBaseIRI: ${METADATA_BASE_IRI:http://localhost/iri/}
+    # Jena's TDB2 database path
+    datasetPath: ${DATASET_PATH:/Users/anton/git/rdm/fairspace/projects/saturn/data/db}
```
Comment (ewelinajozwik) [inline projects/saturn/src/main/resources/application.yaml]: @tgreenwood please change this, `datasetPath: ${DATASET_PATH:data/db}` should work

**Comment by tgreenwood** [comment]
Update readme (at least the part about fuseki)

File: projects/saturn/src/main/resources/application.yaml (hunk 4 above)
Comment (tgreenwood) [reply projects/saturn/src/main/resources/application.yaml]: done

**Review by ewelinajozwik — APPROVED** [review]

**Pull request merged**